### PR TITLE
clarify ClusterSingletonProxy role docs

### PR DESCRIPTION
**Pull request by patriknw**
Reported in https://discuss.lightbend.com/t/singleton-cluster-a-multiple-roles-with-proxy-b-prestart/6381/3

### CODE REVIEW

File: akka-cluster-tools/src/main/resources/reference.conf [hunk 1]
```diff
@@ -197,8 +197,9 @@ akka.cluster.singleton-proxy {
   # The actor name of the singleton actor that is started by the ClusterSingletonManager
   singleton-name = ${akka.cluster.singleton.singleton-name}
   
-  # The role of the cluster nodes where the singleton can be deployed. 
-  # If the role is not specified then any node will do.
+  # The role of the cluster nodes where the singleton can be deployed.
+  # Corresponding to the role used by the `ClusterSingletonManager`. If the role is not
+  # specified it's a singleton among all nodes in the cluster.
```
Comment (johanandren) [inline akka-cluster-tools/src/main/resources/reference.conf:202]: Note that it must be the same for the singleton and the proxy as well?

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-cluster-tools/src/main/resources/reference.conf (hunk 1 above)
Comment (patriknw) [reply akka-cluster-tools/src/main/resources/reference.conf]: ok, added some more clarification

Comment (johanandren) [reply akka-cluster-tools/src/main/resources/reference.conf]: 👍

**Review by chbatey — APPROVED** [review]

**Pull request merged**